tests/AppFixture.js

```diff
@@ -0,0 +1,2 @@
+baseline = /\d+ variables et \d+ paramètres référencées pour modéliser le système socio-fiscal français\./
```

Comment:
It seems too rigid, we don't need to test so precisely.

`/\d+ variables et \d+ paramètres référencées/` would probably be enough and avoid reworking on the test if we change the second part a little.